### ISSUE TITLE
Admin looses the permission to see all the users when that permission is removed to default

### ISSUE THREAD

**Issue by puneethpotu** [getredash/redash#696]
When i remove the ```list_users``` permission to default, Admin looses the access to see the list of users as well.

Workaround
When i add the ```list_users``` explicitly to the admin and remove it from Default group then, i can see the list of users.

Are the ```Admin permissions = Default permissions + Some more Admin privileges```
or
They both are independent of each other?

**Comment by arikfr** [comment]
A user's permissions are the union of all the permissions of the groups he's member of. An admin is a user who is a member of a group that has the `admin` permission (the group name itself is meaningless).

In default setup, we have the default group that has all the "standard" permissions and the admin group that has the `admin` permission. So it results in admin permissions = default permissions + admin permission.

To summarize: if you want only admins to be able to list users, add the `list_users` permission to the admin group and remove it from the default group.
